backend/src/test/java/gov/cdc/usds/simplereport/service/TestResultUploadServiceTest.java

```
@@ -325,19 +188,29 @@ void uploadService_getUploadSubmission_fileInvalidData() {
     List<TestResultUpload> result = sut.processResultCSV(invalidInput);
 
     // THEN
+    assertEquals(1, result.size());
     assertThat(result.get(0).getStatus()).isEqualTo(FAILURE);
     assertThat(result.get(0).getErrors()).hasSize(1);
     assertThat(result.get(0).getErrors()[0].getMessage()).isEqualTo("my lovely error message");
   }
 
   @Test
   @SliceTestConfiguration.WithSimpleReportStandardUser
-  void uploadService_getUploadSubmission_processingModeCode_NotSet() {
+  void uploadService_getUploadSubmission_processingModeCode_NotSet() throws Exception {
     // GIVEN
     ArgumentCaptor<byte[]> fileContentCaptor = ArgumentCaptor.forClass(byte[].class);
     InputStream input = loadCsv("testResultUpload/test-results-upload-valid.csv");
-    UploadResponse response = buildUploadResponse();
-    when(dataHubMock.uploadCSV(any())).thenReturn(response);
+    UploadResponse successfulResponse = buildSuccessfulUploadResponse();
+    var tokenResponse = new TokenResponse();
+    tokenResponse.setAccessToken("fake-rs-access-token");
+
+    when(tokenAuthMock.createRSAJWT(anyString(), anyString(), any(Date.class), anyString()))
+        .thenReturn("fake-rs-sender-token");
+    when(orgServiceMock.getCurrentOrganization()).thenReturn(factory.saveValidOrganization());
+    when(bulkUploadFhirConverterMock.convertToFhirBundles(any(), any()))
+        .thenReturn(new FHIRBundleRecord(List.of("a", "b", "c"), new HashMap<>()));
+    when(dataHubMock.fetchAccessToken(anyString())).thenReturn(tokenResponse);
+    when(dataHubMock.uploadCSV(any())).thenReturn(successfulResponse);
```

Any tests where these lines were added were silently failing on the universal pipeline since they were throwing an exception that we ignored. With the added exception propagating in this PR, I had to set up these mocks to prevent that exception from being thrown.